app/controllers/root_controller.rb

```diff
@@ -273,10 +259,42 @@ def treat_as_standard_html_request?
     !request.format.json? and !request.format.print? and !request.format.video?
   end
 
-  def local_transaction_details(artefact, authority_slug, snac)
-    raise RecordNotFound if snac.blank? && authority_slug.present?
+  def local_transaction_details(artefact, authority_slug)
```

Comment:
Instead of putting this in the controller, we could use a separate object to construct the local links manager request and add the details into the artefact, as [finder-frontend does](https://github.com/alphagov/finder-frontend/blob/master/app/lib/finder_api.rb).